pkg/operator/operator.go

```diff
@@ -130,10 +136,24 @@ func NewOperator(ctx context.Context, operator *operator.Operator) (context.Cont
 		log.FromContext(ctx).WithValues("kube-dns-ip", kubeDNSIP).V(1).Info("discovered kube dns")
 	}
 
+	//v2
+	cfg := lo.Must(configV2.LoadDefaultConfig(ctx))
```

Comment:
We should define and use the default retry logic https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/retries-timeouts/#standard-retryer